src/pages/Examples/Topology/topology-view.js

```diff
@@ -9,7 +9,7 @@ export const TopologyView = ({
   setSelectionMode,
   toggleChecked
 }) => (
-  <div className="font-inter flex leading-1.21rel">
+  <div className="flex leading-1.21rel">
```

Comment:
Can we move the example to a storybook ?

Reply:
I'll do it in the follow up PR